cdc/http_status.go

```diff
@@ -38,7 +38,7 @@ import (
 	"go.uber.org/zap"
 )
 
-func (s *Server) startStatusHTTP() error {
+func (s *Server) createStatusHTTP() error {
```

Comment:
Could you define a StatusServer struct to decoupling createStatusHTTP from Server?

```go
statusServer := createStatusServer()
s.statusServer = statusServer
```